repos/system_upgrade/el7toel8/actors/ipuworkflowconfig/libraries/library.py

```diff
@@ -12,3 +13,26 @@ def get_env_vars():
     available after reboot.
     """
     return [EnvVar(name=k, value=v) for (k, v) in os.environ.items() if k.startswith('LEAPP_') and k not in ENV_IGNORE]
+
+
+def get_os_release(path):
+    """Retrieve data about System OS release from provided file."""
```

Comment:
Docstrings should be like this in our project; from what I have learnt recently:

"""
text
"""

Reply:
From [Contribution guidelines](https://github.com/oamg/leapp-repository/blob/master/CONTRIBUTING.md):
> Follow PEP 257 - Docstring Conventions

And from [PEP 257](https://www.python.org/dev/peps/pep-0257/#one-line-docstrings) for One-line Docstrings:
> The closing quotes are on the same line as the opening quotes. This looks better for one-liners.

Reply:
Then I probably misunderstood it in my previous case. Fair enough...